x-pack/test/functional/apps/ml/data_visualizer/index_data_visualizer_actions_panel.ts

```diff
@@ -58,5 +58,32 @@ export default function ({ getService }: FtrProviderContext) {
         await ml.jobWizardAdvanced.assertDatafeedQueryEditorValue(advancedJobWizardDatafeedQuery);
       });
     });
+
+    describe('view in discover page action', function () {
+      const savedSearch = 'ft_farequote_kuery';
```

Comment:
This test suite is not creating the saved search used here. It's working as long as it's executed after the `index_data_visualizer`, which creates it, but it's not working on its own. We need to add `await ml.testResources.createSavedSearchFarequoteKueryIfNeeded();` to `before` in order to make this suite independent.

Reply:
Updated [here](https://github.com/elastic/kibana/pull/89471/commits/96ff5ac3cc84ac7cc724c7cd793b40cfbe4b2b60)